examples/next-ui/README.md

```diff
@@ -0,0 +1,22 @@
+# next-ui example
+
+Using [next-ui](https://nextui.org/) Library with remix
+
+## Preview
+
+Open this example on [CodeSandbox](https://codesandbox.com):
+
+
+[![Open in CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/remix-run/remix/tree/main/examples/next-ui)
+
+## Example
+
+Describe the example and how it demonstrates solving the problem. Reference any relevant files/dependencies if needed.
```

Comment:
Could you fill this in with some notes about what this example is demonstrating?

Reply:
Sure 👍🏻